ruby/.standard.yml

```diff
@@ -1,5 +1,9 @@
 ---
 parallel: true
 ruby_version: '2.7'
+plugins:
+  - standard-minitest
+  # - standard-thread_safety
```

Comment:
Do you want to delete this line rather than comment it out?

Reply:
No, because I don't want to forget it when I get to the Ruby stuff (in May?).